src/containers/ChainOverview/Chart.tsx

```diff
@@ -94,7 +102,7 @@ export default function ChainLineBarChart({
 					  }
 					: {}),
 				markLine: {},
-				data: chartData[stack] ?? []
+				data: (chartData[stack] ?? []).map(([timestamp, value]) => [timestamp * 1000, value])
```

Comment:
we don't want iterate over chart data again in this component, the dates should already be formatted in milliseconds